kfs-core/src/main/java/edu/arizona/kfs/gl/batch/service/impl/BudgetAdjustmentCashTransferServiceImpl.java

```diff
@@ -0,0 +1,596 @@
+                        	 saveDocNumber = originEntry.getDocumentNumber();
+                         }                         
+                         originEntry.setTransactionLedgerEntrySequenceNumber(new Integer(maxSequenceId));
+                         
+                         BudgetAdjustmentTransaction ba = new BudgetAdjustmentTransaction(originEntry);                         
+                    	 try {
+                             budgetAdjustmentTransactionDao.save(ba);
+                             reportBudgetAdjustDocLoaded++;
+                         }
+                         catch (RuntimeException re) {
+                        	 //error adding budget adjustment record, write error and continue
+                        	 LOG.error("generateBudgetAdjustmentCashTransferTransactions exception: " + re.getMessage());
+                             Message errorMsg = new Message("Error adding budget adjustment record for this record.", Message.TYPE_FATAL);
+                             reportWriterService.writeError(originEntry, errorMsg);
+                             createOutputEntry(GLEN_RECORD, OUTPUT_ERR_FILE_ps);
+                             reportBudgetAdjustDocErrors++;
+                             continue;
+                         }
+                     }
+                     
+                 }
+             }
+             INPUT_GLE_FILE_br.close();
+             INPUT_GLE_FILE.close();
+             OUTPUT_ERR_FILE_ps.close();
+             reportWriterService.writeStatisticLine("SEQUENTIAL RECORDS READ                        %,9d", line);
+             reportWriterService.writeStatisticLine("GLBA RECORDS INSERTED (GL_BUDGET_ADJUST_TRN_T) %,9d", reportBudgetAdjustDocLoaded);
+             reportWriterService.writeStatisticLine("ERROR RECORDS WRITTEN                          %,9d", reportBudgetAdjustDocErrors);
+         }
+         catch (IOException e) {
+             throw new RuntimeException(e);
+         }         
+    }
+
+    /**
+     * Reads budget adjustment transactions from holding table GL_BUDGET_ADJUST_TRN_T and generates/builds 
+     * a file of budget adjustment cash transfer entries for posting to the General Ledger.
+     */
+    @Override
+    public void generateBudgetAdjustmentCashTransferTransactions() {
+    	LOG.debug("generateBudgetAdjustmentCashTransferTransactions() started");
+        
+        Date executionDate = dateTimeService.getCurrentSqlDate();
+        Date runDate = new Date(runDateService.calculateRunDate(executionDate).getTime());
+        int reportBudgetAdjustDocLoaded = 0;
+        int reportOriginEntryGenerated = 0;
+    	   	
+    	 try {
+    		 PrintStream OUTPUT_GLE_FILE_ps = new PrintStream(batchFileDirectoryName + File.separator + GeneralLedgerConstants.BatchFileSystem.CASH_TRANSFER_TRANSACTIONS_OUTPUT_FILE + GeneralLedgerConstants.BatchFileSystem.EXTENSION);
+    		     		              
+             //retrieve unique document numbers list             
```

Comment:
comment not needed, obvious from the method name what we're retrieving. Comment should state the purpose maybe of retrieving it?